include/openmc/dagmc.h

```diff
@@ -9,11 +9,15 @@ extern "C" const bool dagmc_enabled;
 #ifdef DAGMC
 
 #include "DagMC.hpp"
-#include "openmc/cell.h"
-#include "openmc/surface.h"
+#include "openmc/xml_interface.h"
+#include "openmc/position.h"
 
 namespace openmc {
 
+extern moab::DagMC::RayHistory history;
+extern Direction last_dir;
```

Comment:
Global variables should be in a [sub-namespace](http://docs.openmc.org/en/latest/devguide/styleguide.html#naming).

Reply:
`model` or `simulation` would you say?

Reply:
I went with `simulation` as that's when they are used, let me know if I'm off on that.

Reply:
Yup, `simulation` makes the most sense